changelogs/fragments/9691-keycloak-module-utils-replace-missing-return-in-get_role_composites.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - module_utils/identity/keycloak - replaces missing return in get_role_composites method which caused it to return None instead of composite roles (https://github.com/ansible-collections/community.general/issues/9678)
```

Comment:
```suggestion
  - keycloak module utils - replaces missing return in get_role_composites method which caused it to return None instead of composite roles (https://github.com/ansible-collections/community.general/issues/9678, https://github.com/ansible-collections/community.general/pull/9691).
```